region/brinstar/red.json

```diff
@@ -817,7 +832,8 @@
                     {"canComeInCharged": {
                       "fromNode": 1,
                       "framesRemaining": 0,
-                      "shinesparkFrames": 150
+                      "shinesparkFrames": 150, 
+                      "excessShinesparkFrames": 57
```

Comment:
If the spark ends early, close to the right door, is there time to respond before falling into spikes? If not, I dont think we need to model it now, but a devnote might be beneficial